.github/DISCUSSION_TEMPLATE/issue-triage.yml

```
@@ -93,9 +98,9 @@ body:
       required: false
   - type: textarea
     attributes:
-      label: Ghostty Configuration
+      label: Ghostty Configuration (Minimal)
       description: |
-        Please provide the minimum configuration needed to reproduce this issue.  If you cannot determine this, paste the output of `ghostty +show-config` here.
+        Please provide the **minimum** configuration needed to reproduce this issue.  If you cannot determine this, paste the contents of your Ghostty configuration file here.
```

I feel like a “what a minimum configuration even is” would be helpful.  I can't think of any way to elaborate without wasting too much space, though.
More emphasis on the last part being a last resort option would be nice too (which I include in the suggestion, though it's not too obvious in the diff), since it almost seems like everybody is going down the path of least resistance and just pasting `ghostty +show-config` at the moment.
```suggestion
        Please provide the **minimum** configuration needed to reproduce this issue.  If you can still reproduce the issue with one of the lines removed, do not include that line.  If and **only** if you are not able to determine this, paste the contents of your Ghostty configuration file here.
```

---

If others think this is necessary, I'm happy to add it.  I feel like we should expect _some_ level of competency for users reporting issues, and minimum configuration needed for reproduction seems like a pretty low bar, IMO

---

I dunno, I just haven't seen all that many minimum configurations floating around, almost everyone seems to include their entire configuration file or the entire output of `ghostty +show-config` even though it can be reproduced with no configuration.